examples/intel/user/password_breached.go

```diff
@@ -23,15 +23,16 @@ func main() {
 	})
 
 	// Set the password you would like to check
+	// Observe proper safety with passwords, do not check them into source control etc.
 	password := "mypassword"
-	// Calculate its hash, it could be sha256 or sha1
+	// Calculate it's hash, it could be sha256, sha512 or sha1
```

Comment:
I meant "calculate password hash"

Reply:
Actually "its" is actually correct for the possessive. "It's" is a replacement for "it is"